package.json

```diff
@@ -141,6 +148,7 @@
     "stylelint-no-unsupported-browser-features": "^5.0.3",
     "stylelint-order": "^5.0.0",
     "stylelint-use-logical-spec": "^3.2.2",
+    "tsconfig-paths-webpack-plugin": "^4.1.0",
```

Comment:
Why do we need it for webpack?

Reply:
because we use it Storybook. Will be removed when we will move to vite instead of webpack